### DISCUSSION TITLE
Default config only displays a very small number of posts in sidebar?

### DISCUSSION THREAD

**Discussion by cmdcolin** [facebook/docusaurus#4522] [General] [answered]
The current "Recent posts" list is pretty small (5 entries) and doesn't really give much indication if you have more than that

The "Blog archive" style model from Blogspot might be good, with a calendar listing that is expandable by year and month, but I'm not really tied to that model.

My concern is mostly that someone might see recent posts only containing 5 entries and become bored, not even noticing the "Older entries" button at the bottom of the page

**Comment by cmdcolin** [comment]
I found there is the

          blogSidebarCount: 'ALL',


But it also said that is deprecated in the docs?

**Comment by slorber** [comment]
We have an issue for a blog archive pagz

Where have you seen blogSidebarCount is deprecated?

Le sam. 27 mars 2021 à 00:22, Colin Diesh ***@***.***> a
écrit :

**Reply by cmdcolin** [reply]
The deprecation is listed here https://docusaurus.io/docs/migration/manual#blogsidebarcount and I got to that page searching "blogSidebarCount" in the search box

**Comment by slorber** [comment, marked as answer]
This is a mistake, it has been ported to v2 😉

Le sam. 27 mars 2021 à 04:12, Colin Diesh ***@***.***> a
écrit :

**Reply by cmdcolin** [reply]
Gotcha, will look forward to updates :)

**Reply by cmdcolin** [reply]
If possible, might want to update the docs for this?

**Reply by slorber** [reply]
sure, do you want to send a PR to the migration guide doc?
